docs/user-guide/connectors/custom-connectors.rst

```diff
@@ -31,6 +31,17 @@ To send a message, you would run a command like:
 
 where ``myio`` is the name of your component.
 
+You can also send extra information from your front-end to your
+action server, with it you can use, e.g. name, locations, or tokens
+in our actions. To send a message with ``metadata`` you would run
+a command like:
+
+.. code-block:: bash
+
+    curl -XPOST http://localhost:5000/webhooks/myio/webhook \
+      -d '{"sender": "user1", "message": "here is my token: <token>"}, "metadata: {"type": "token"}' \
```

Comment:
This looks like it might not be formatted correctly, think that parenthesis after <token> is in the wrong place?

Reply:
Yes! I have corrected it